packages/cli/source/manifest/compiler.ts

```diff
@@ -0,0 +1,16 @@
+import yaml from "js-yaml";
+import { glob } from "glob";
+import fs from "fs";
+
+import * as paths from "../config/paths";
+
+const compiler = () => {
+    glob(paths.MANIFEST_DIRECTORY + "**/*.{yml, yaml}", function (err, files) {
```

Comment:
1. Use arrow functions.
2. Rename `err` to `error`.
3. Add type annotations to the function.